en/docs/reference/config-catalog.md

```diff
@@ -725,193 +700,176 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
+                                  <span class="param-name-wrap"> <code>enable_user_claims</code> </span>
                                 </div>
                                 <div class="param-info">
                                     <div>
                                         <p>
                                             <span class="param-type string"> string </span>
-
+                                            
                                         </p>
                                         <div class="param-default">
                                             <span class="param-default-value">Default: <code>FALSE</code></span>
                                         </div>
                                         <div class="param-possible">
-                                            <span class="param-possible-values">Possible Values:
-                                                <code>TRUE</code></span>
+                                            <span class="param-possible-values">Possible Values: <code>TRUE</code></span>
                                         </div>
                                     </div>
                                     <div class="param-description">
                                         <p>THe default user claims will be added to the JWT</p>
                                     </div>
                                 </div>
-                            </div>
-                            <div class="param">
+                            </div><div class="param">
                                 <div class="param-name">
-                                    <span class="param-name-wrap"> <code>claims_extractor_impl</code> </span>
+                                  <span class="param-name-wrap"> <code>claims_extractor_impl</code> </span>
                                 </div>
                                 <div class="param-info">
                                     <div>
                                         <p>
                                             <span class="param-type string"> string </span>
-
+                                            
                                         </p>
                                         <div class="param-default">
                                             <span class="param-default-value">Default: <code></code></span>
                                         </div>
-
+                                        
                                     </div>
                                     <div class="param-description">
-                                        <p>You can define the custom claim retiriver implmentation by adding the
-                                            following
-                                            apim.jwt.enable_user_claims=true.
+                                        <p>You can define the custom claim retiriver implmentation by adding the following 

```

Comment:
Shall we fix these?
"retriever"
"implementation"

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3004